GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Ariane.cfg

```diff
@@ -268,34 +270,29 @@
     @manufacturer = Airbus Defence and Space
     @description = The second stage of the Ariane 5 launch vehicle family (ESC-A variant). Contains guidance capability.
 
-    @mass = 4.375
+    @mass = 4.45
     @crashTolerance = 14
     @breakingForce = 250
     @breakingTorque = 250
     @maxTemp = 773.15
     %skinMaxTemp = 873.15
 
-    @MODULE[ModuleRCS]
+    //  56 N Gas Hydrogen (GH2) attitude control thrusters for each axis (using liquid hydrogen directly for now).
+
+    @MODULE[ModuleRCS*]
     {
-        @thrusterPower = 0.4
+        @thrusterPower = 0.056
         !resourceName = NULL
 
         PROPELLANT
         {
-            name = Hydrazine
+            name = LqdHydrogen
             ratio = 1.0
         }
 
-        PROPELLANT
-        {
-            name = Nitrogen
-            ratio = 10.0
-            ignoreForIsp = True
-        }
-
         @atmosphereCurve
         {
-            @key,0 = 0 216
+            @key,0 = 0 180
```

Comment:
I am awaiting the RealFuels release for KSP 1.2. @Starwaster mentioned that he will add the capability for cryogenics themselves to produce gas products (LqdHydrogen --> Hydrogen) without the need of any converter modules.

This could also come handy with CMES to create cryocoolers for the Mars Transfer Vehicle.